sentry/migrations/20190806011140_initial-tables/up.sql

```diff
@@ -3,37 +3,33 @@ CREATE TABLE channels
     id             VARCHAR(66)              NOT NULL,
     creator        VARCHAR(255)             NOT NULL,
     deposit_asset  VARCHAR(42)              NOT NULL,
-    deposit_amount VARCHAR(255)             NOT NULL, -- @TODO change the deposit to BigNum compatible field
+    deposit_amount VARCHAR(255)             NOT NULL,
     valid_until    TIMESTAMP WITH TIME ZONE NOT NULL,
     spec           JSONB                    NOT NULL,
 
     PRIMARY KEY (id)
 );
 
-CREATE INDEX idx_valid_until ON channels (valid_until);
-CREATE INDEX idx_spec ON channels ((spec -> 'validator' ->> 'id'));
```

Comment:
We should see if this is even possible with value in array without using heavy indices like these:
https://www.postgresql.org/docs/9.4/datatype-json.html#JSON-CONTAINMENT